zulip_bots/zulip_bots/bots/xkcd/xkcd.py

```diff
@@ -111,7 +116,7 @@ def fetch_xkcd_query(mode, comic_id=None):
 
         xkcd_json = fetched.json()
     except requests.exceptions.ConnectionError as e:
-        logging.warning(e)
+        logging.warning(str(e))
         raise
```

Comment:
Okay, I know what's causing this and I think it would be better to do the following:
```
logging.exception("Connection Error")
```
`logging.exception` automatically takes care of including the exception info in the logged message! :)

Reply:
@eeshangarg OK, changed. Thanks!